components/colorwheel/package.json

```diff
@@ -18,12 +18,14 @@
   },
   "peerDependencies": {
     "@spectrum-css/colorhandle": "^2.0.0",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/colorarea": "^1.0.29",
+    "@spectrum-css/tokens": "^7.0.0"
   },
   "devDependencies": {
-    "@spectrum-css/colorhandle": "^2.0.14",
-    "@spectrum-css/component-builder": "^4.0.0",
-    "@spectrum-css/vars": "^8.0.3",
+    "@spectrum-css/colorhandle": "^2.0.11",
+    "@spectrum-css/colorarea": "^1.0.29",
```

Comment:
I added ColorArea as a dependency so it could be displayed within the ColorWheel (see screenshot). This is included in the XD file and is closely related to the token (--spectrum-color-wheel-color-area-margin) but I was not sure if we wanted to have this added or not. Please let me know if we'd prefer to not include this.

Reply:
yes this makes sense. They are both used for this component and I can't imagine an instance where we would have a color wheel without the handle.